frontend/admin/src/js/components/searchRequest/SearchRequestTable.tsx

```diff
@@ -103,21 +149,21 @@ export const SearchRequestTable: React.FunctionComponent<
       },
       {
         Header: intl.formatMessage({
-          defaultMessage: "Department",
-          id: "i3C5Hn",
+          defaultMessage: "Email",
+          id: "RwYcDw",
           description:
-            "Title displayed on the search request table department column.",
+            "Title displayed on the search requests table email column.",
         }),
-        accessor: ({ department }) => department?.name?.[locale],
+        accessor: "email",
       },
       {
         Header: intl.formatMessage({
-          defaultMessage: "Email",
-          id: "hiZAeF",
+          defaultMessage: "Department",
+          id: "i3C5Hn",
           description:
-            "Title displayed on the search request table email column.",
+            "Title displayed on the search request table department column.",
         }),
-        accessor: "email",
+        accessor: ({ department }) => getLocalizedName(department?.name, intl),
```

Comment:
changing from 
search request table email column
to
search _request**s**_ table email column
disconnected the intl, please update fr.json